<!DOCTYPE html>
<html lang="en" dir="ltr" class="client-nojs">
<head>
<meta charset="UTF-8" />
<title>Team:HUST-China/Engineering - 2021.igem.org</title>
<meta name="generator" content="MediaWiki 1.24.1" />
<link rel="shortcut icon" href="/favicon.ico" />
<link rel="search" type="application/opensearchdescription+xml" href="/wiki/opensearch_desc.php" title="2021.igem.org (en)" />
<link rel="alternate" hreflang="x-default" href="/Team:HUST-China/Engineering" />
<link rel="copyright" href="https://creativecommons.org/licenses/by/3.0/" />
<link rel="alternate" type="application/atom+xml" title="2021.igem.org Atom feed" href="/wiki/index.php?title=Special:RecentChanges&amp;feed=atom" />
<link rel="stylesheet" href="https://2021.igem.org/wiki/load.php?debug=false&amp;lang=en&amp;modules=mediawiki.legacy.commonPrint%2Cshared%7Cmediawiki.skinning.content.externallinks%7Cmediawiki.skinning.interface%7Cmediawiki.ui.button%7Cskins.igem.styles&amp;only=styles&amp;skin=igem&amp;*" />
<!--[if IE 6]><link rel="stylesheet" href="/wiki/skins/Igem/IE60Fixes.css?303" media="screen" /><![endif]-->
<!--[if IE 7]><link rel="stylesheet" href="/wiki/skins/Igem/IE70Fixes.css?303" media="screen" /><![endif]--><meta name="ResourceLoaderDynamicStyles" content="" />
<style>a:lang(ar),a:lang(kk-arab),a:lang(mzn),a:lang(ps),a:lang(ur){text-decoration:none}
/* cache key: 2021_igem_org:resourceloader:filter:minify-css:7:329647e7075f31ea1bfe23cc06e29c9e */</style>
<script src="https://2021.igem.org/wiki/load.php?debug=false&amp;lang=en&amp;modules=startup&amp;only=scripts&amp;skin=igem&amp;*"></script>
<script>if(window.mw){
mw.config.set({"wgCanonicalNamespace":"","wgCanonicalSpecialPageName":false,"wgNamespaceNumber":0,"wgPageName":"Team:HUST-China/Engineering","wgTitle":"Team:HUST-China/Engineering","wgCurRevisionId":495843,"wgRevisionId":495843,"wgArticleId":1486,"wgIsArticle":true,"wgIsRedirect":false,"wgAction":"view","wgUserName":null,"wgUserGroups":["*"],"wgCategories":[],"wgBreakFrames":false,"wgPageContentLanguage":"en","wgPageContentModel":"wikitext","wgSeparatorTransformTable":["",""],"wgDigitTransformTable":["",""],"wgDefaultDateFormat":"dmy","wgMonthNames":["","January","February","March","April","May","June","July","August","September","October","November","December"],"wgMonthNamesShort":["","Jan","Feb","Mar","Apr","May","Jun","Jul","Aug","Sep","Oct","Nov","Dec"],"wgRelevantPageName":"Team:HUST-China/Engineering","wgIsProbablyEditable":false,"wgRestrictionEdit":[],"wgRestrictionMove":[]});
}</script><script>if(window.mw){
mw.loader.implement("user.options",function($,jQuery){mw.user.options.set({"ccmeonemails":0,"cols":80,"date":"default","diffonly":0,"disablemail":0,"editfont":"default","editondblclick":0,"editsectiononrightclick":0,"enotifminoredits":0,"enotifrevealaddr":0,"enotifusertalkpages":1,"enotifwatchlistpages":1,"extendwatchlist":0,"fancysig":0,"forceeditsummary":0,"gender":"unknown","hideminor":0,"hidepatrolled":0,"imagesize":2,"math":1,"minordefault":0,"newpageshidepatrolled":0,"nickname":"","norollbackdiff":0,"numberheadings":0,"previewonfirst":0,"previewontop":1,"rcdays":7,"rclimit":50,"rows":25,"showhiddencats":0,"shownumberswatching":1,"showtoolbar":1,"skin":"igem","stubthreshold":0,"thumbsize":5,"underline":2,"uselivepreview":0,"usenewrc":1,"watchcreations":1,"watchdefault":1,"watchdeletion":0,"watchlistdays":3,"watchlisthideanons":0,"watchlisthidebots":0,"watchlisthideliu":0,"watchlisthideminor":0,"watchlisthideown":0,"watchlisthidepatrolled":0,"watchmoves":0,"watchrollback":0,
"wllimit":250,"useeditwarning":1,"prefershttps":1,"language":"en","variant-gan":"gan","variant-iu":"iu","variant-kk":"kk","variant-ku":"ku","variant-shi":"shi","variant-sr":"sr","variant-tg":"tg","variant-uz":"uz","variant-zh":"zh","searchNs0":true,"searchNs1":false,"searchNs2":false,"searchNs3":false,"searchNs4":false,"searchNs5":false,"searchNs6":false,"searchNs7":false,"searchNs8":false,"searchNs9":false,"searchNs10":false,"searchNs11":false,"searchNs12":false,"searchNs13":false,"searchNs14":false,"searchNs15":false,"variant":"en"});},{},{});mw.loader.implement("user.tokens",function($,jQuery){mw.user.tokens.set({"editToken":"+\\","patrolToken":"+\\","watchToken":"+\\"});},{},{});
/* cache key: 2021_igem_org:resourceloader:filter:minify-js:7:c2a6e0f393444edcfe542f485c3692c2 */
}</script>
<script>if(window.mw){
mw.loader.load(["mediawiki.page.startup","mediawiki.legacy.wikibits","mediawiki.legacy.ajax"]);
}</script>
</head>
<body class="mediawiki ltr sitedir-ltr ns-0 ns-subject page-Team_HUST-China_Engineering skin-igem action-view">

        <script type='text/javascript'         src ='/common/tablesorter/jquery.tablesorter.min.js'></script>
        <link rel='stylesheet' type='text/css' href='/common/tablesorter/themes/groupparts/style.css' />
        <link rel='stylesheet' type='text/css' href='/common/table_styles.css' />

        <script type='text/javascript'         src ='/wiki/skins/Igem/resources/2021_skin.js'></script>


    <div id='globalWrapper'>
        <div id='top_menu_under' class='noprint'></div>
        <div id='top_menu_14' class='noprint'>Loading menubar.....</div> <!-- Will be replaced with the jQuery.load -->
		<script>jQuery('#top_menu_14').load('https://2021.igem.org/cgi/top_menu_14/menubar_reply.cgi',
						    {   t:"Team%3AHUST-China%2FEngineering",
							a:"View+%2FTeam%3AHUST-China%2FEngineering++View source+%2Fwiki%2Findex.php%3Ftitle%3DTeam%3AHUST-China%2FEngineering%26action%3Dedit++History+%2Fwiki%2Findex.php%3Ftitle%3DTeam%3AHUST-China%2FEngineering%26action%3Dhistory++Page+%2FTeam%3AHUST-China%2FEngineering++Discussion+%2Fwiki%2Findex.php%3Ftitle%3DTalk%3ATeam%3AHUST-China%2FEngineering%26action%3Dedit%26redlink%3D1++" });
		</script>

        <!-- Content div contains HQ_page for HQ styles, Logo and title div, and USER CONTENT -->
		<div id="content" class="mw-body" role="main">
		    <a id="top"></a>

            <div id="top_title">
                <div class="logo_2021">
                    <a href="https://2021.igem.org">
                    <img src="./statics/pics/Engineering/HQ_page_logo.jpg" width="100px">
                    </a>
                </div>

 		        <h1 id="firstHeading" class="firstHeading">
		            <span dir="auto">Team:HUST-China/Engineering</span>
		        </h1>
            </div>

            <div id="HQ_page">
                <div id="bodyContent">
		            <div id="mw-content-text" lang="en" dir="ltr" class="mw-content-ltr"><p>

<head>
    <meta charset="utf-8" />
    <meta content="width=device-width,initial-scale=1" name="viewport" />
    <link href="https://2021.igem.org/Template:HUST-China/css/contentCSS?action=raw&amp;ctype=text/css"
        rel="stylesheet" />
    <title>Engineering success | iGEM HUST-China</title>
    <link href="https://2021.igem.org/Template:HUST-China/css/contentCSS?action=raw&amp;ctype=text/css"
        rel="stylesheet" />
</head>

<body>
    <!-- # TODO: #6 Fix table caption font-->
    <!-- # TODO: #7 Fix citations links font size-->
    <nav class="navbar navbar-expand-xl fixed-top">
        <div class="container d-flex justify-content-between"><a class="navbar-brand"
                href="https://2021.igem.org/Team:HUST-China"><i class="navbar-logo-left"></i><span>HUST-China</span></a>
            <button aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"
                class="navbar-toggler" data-target="#navbarNav" data-toggle="collapse" type="button"><span
                    class="navbar-toggler-icon"></span>
            </button>
            <div class="collapse navbar-collapse" id="navbarNav">
                <ul class="navbar-nav ml-auto">
                    <li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true"
                            class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarProjectDropdown"
                            role="button">Project</a>
                        <div aria-labelledby="navbarProjectDropdown" class="dropdown-menu"><a class="dropdown-item"
                                href="Description.html">Description</a><a
                                class="dropdown-item" href="Design.html">Design</a><a
                                class="dropdown-item"
                                href="Proof_Of_Concept.html">Proof of
                                Concept</a><a class="dropdown-item"
                                href="Safety.html">Safety</a><a class="dropdown-item"
                                href="Implementation.html">Proposed
                                Implementation</a><a class="dropdown-item"
                                href="Contribution.html">Contribution</a>
                        </div>
                    </li>
                    <li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true"
                            class="nav-link dropdown-toggle" data-toggle="dropdown" href="#"
                            id="navbarHuman PracticesDropdown" role="button">Human Practices</a>
                        <div aria-labelledby="navbarHuman PracticesDropdown" class="dropdown-menu"><a
                                class="dropdown-item"
                                href="Human_Practices.html">Overview</a><a
                                class="dropdown-item"
                                href="Human_Practices_01.html">Part
                                1</a><a class="dropdown-item"
                                href="Human_Practices_02.html">Part
                                2</a><a class="dropdown-item"
                                href="Collaborations.html">Part
                                3</a><a class="dropdown-item"
                                href="Education.html">Part 4</a>
                        </div>
                    </li>
                    <li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true"
                            class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarModelingDropdown"
                            role="button">Modeling</a>
                        <div aria-labelledby="navbarModelingDropdown" class="dropdown-menu"><a class="dropdown-item"
                                href="Model.html">Overview</a><a class="dropdown-item"
                                href="Gene_Pathway.html">Gene
                                Pathway</a><a class="dropdown-item"
                                href="Dye_color_forecast.html">Dye color
                                forecast</a><a class="dropdown-item"
                                href="Micronucleus_Counter.html">Micronucleus
                                Counter</a></div>
                    </li>
                    <li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true"
                            class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarProductsDropdown"
                            role="button">Products</a>
                        <div aria-labelledby="navbarProductsDropdown" class="dropdown-menu"><a class="dropdown-item"
                                href="Products.html">Overview</a><a
                                class="dropdown-item"
                                href="Software.html">Software</a><a
                                class="dropdown-item"
                                href="Hardware.html">Hardware</a><a
                                class="dropdown-item" href="Another_Area.html">Another
                                Area</a></div>
                    </li>
                    <li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true"
                            class="nav-link dropdown-toggle" data-toggle="dropdown" href="#"
                            id="navbarExperimentsDropdown" role="button">Experiments</a>
                        <div aria-labelledby="navbarExperimentsDropdown" class="dropdown-menu"><a class="dropdown-item"
                                href="Experiments.html">Overview</a><a
                                class="dropdown-item" href="Parts.html">Parts</a><a
                                class="dropdown-item"
                                href="Engineering.html">Engineering
                                Success</a><a class="dropdown-item"
                                href="Results.html">Results</a><a class="dropdown-item"
                                href="Protocol.html">Protocol</a><a
                                class="dropdown-item" href="Notebook.html">Notebook</a>
                        </div>
                    </li>
                    <li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true"
                            class="nav-link dropdown-toggle" data-toggle="dropdown" href="#" id="navbarPeopleDropdown"
                            role="button">People</a>
                        <div aria-labelledby="navbarPeopleDropdown" class="dropdown-menu"><a class="dropdown-item"
                                href="Team.html">Team</a><a class="dropdown-item"
                                href="Partnership.html">Partnership</a><a
                                class="dropdown-item"
                                href="Attributions.html">Attributions</a></div>
                    </li>
                    <li class="nav-item dropdown"><a aria-expanded="false" aria-haspopup="true"
                            class="nav-link dropdown-toggle" data-toggle="dropdown" href="#"
                            id="navbarAchievementsDropdown" role="button">Achievements</a>
                        <div aria-labelledby="navbarAchievementsDropdown" class="dropdown-menu"><a class="dropdown-item"
                                href="Award.html">Award</a><a class="dropdown-item"
                                href="https://video.igem.org/w/aMwNT6eBryGs8xG5avXD51">Promotion
                                Video</a><a class="dropdown-item"
                                href="https://igem.org/2021_Judging_Form?id=3711">Judging
                                Form</a></div>
                    </li>
                </ul>
            </div>
        </div>
    </nav>
    <div class="navbar-extra fixed-top"></div>
    <header class="d-flex justify-content-center align-items-center">
        <div class="container">
            <h1 id="Top">Engineering success</h1>
            <p class="lead pl-1"></p>
            <hr class="my-4" />
        </div>
    </header>
    <main>
        <div class="container">
            <div class="row">
                <div class="sidebar col-lg-3">
                    <div class="nav" id="contents">
                        <h5>Contents</h5>
                        <ul></ul>
                    </div>
                </div>
                <div class="content col-lg-9">
                    <article>
                        <h1>Introduction:</h1>
                        <p>We want to build an enzyme system that can produce pigments and degrade them according to our
                            needs. For the same idea, we need the second type of enzyme system to produce short peptides
                            for perm, and it can also degrade short peptides at any time. In order to achieve the goal,
                            we designed multiple devices and four systems. In the stage of pathway construction, we
                            tried a variety of methods, summed up the experience, learned from mistakes, and finally
                            achieved some results. Our exploration and work this year has also provided some valuable
                            experience for the follow-up teams.</p>
                        <h1>Cycle1:Construct by the homologous recombination method</h1>
                        <p>Several homologous fragments are designed to link multiple genes altogether at one time,
                            using seamless cloning, to construct a system with complete function. Some success are
                            gained during construction, like the whole pathway of FMO dimer, while significant
                            difficulties occur as the same time. Lessons are learnt and settlement are designed, in
                            order to overcome the present reality of getting much shorter product than expected, after
                            comparing our onlu success to tons of failure.</p>
                        <div class="modal-btn"><button class="btn btn-warning pull-right" data-target="#ES1"
                                data-toggle="modal" type="button" style="margin: 0 5px">Design</button><button class="btn btn-warning pull-right" data-target="#ES2" data-toggle="modal" type="button" style="margin: 0 5px">Build</button><button class="btn btn-warning pull-right" data-target="#ES3" data-toggle="modal" type="button"  style="margin: 0 5px">Test</button><button class="btn btn-warning pull-right" data-target="#ES4" data-toggle="modal" type="button" style="margin: 0 5px">Learn</button>
                            <div aria-hidden="true" aria-labelledby="ES1Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES1" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES1Label">ES1</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>A multi-functional system can achieve a more convenient effect, so that a
                                                plasmid with multiple devices can individually complete tasks that
                                                require plural mono-functional plasmids. According to this idea, we
                                                connected devices with independent functions, forming systems with
                                                complete functions.</p>
                                            <p>In the process of synthesizing each device, the foundation for the
                                                formation of the final system had been laid. The first homologous
                                                segment was first added to the 3' end of the Panb1-crtE-AOX1 Terminator,
                                                Panb1-α factor-4CL-AOX1 Terminator and to the 5' end of the
                                                Panb1-crtB-AOX1 Terminator, Panb1-α factor-ACC-AOX1 Terminator as the
                                                vector.</p>
                                            <p>The second homologous segment was then added at the 5' end of
                                                Panb1-crtB-AOX1 Terminator, Panb1-α factor-ACC-AOX1 Terminator and at
                                                the 3' end of Panb1-crtI-AOX1 Terminator, Panb1-α factor-CUS-AOX1
                                                Terminator.</p>
                                            <p>Finally, the third segment homologous to the gene on the vector plasmid
                                                was finally added at the back end of the Panb1-crtI-AOX1 Terminator,
                                                Panb1-α factor-CUS-AOX1 Terminator.</p>
                                            <p>Three specific homologous segments were designed for using different PCR
                                                primers to add to different gene segments when constructing whole
                                                systems, which, in principle, ensure the experimental feasibility.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES2Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES2" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES2Label">ES2</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>Using the most efficient in-fusion cloning principle, homologous
                                                recombinase is selected to construct the pathway. We design primers
                                                strictly in accordance with the instructions and implement the operating
                                                procedures: first double-enzyme digestion of the plasmids as the
                                                vectors, then PCR amplified gene fragments, and then gel electrophoresis
                                                to verify the vectors and conduct gel extraction for purification.
                                                Finally, the high-purity vectors and segments are connected by
                                                homologous recombinase to construct a system, and the ligated products
                                                are transformed into E.coli TOP10 F'.</p>
                                            <p>Taking lycopene system as an example: firstly, the whole plasmid
                                                containing Panb1-crtE-AOX1 Terminator was used as a vector through
                                                double digestion, then the gene segments of Panb1-crtB-AOX1 Terminator
                                                and Panb1-crtI-AOX1 Terminator was obtained through PCR. Finally, the
                                                purified vector and the two purified segments were linked using
                                                homologous recombinase. The ligated products were then transformed into
                                                E.coli TOP10 F'. The cloned host was inoculated to LB plate containing
                                                AMP 100mg/mL for screening. After a single colony grew, the spot was
                                                picked for colony PCR verification. If the target band is observed, mix
                                                the bacteria solution with a 1:1 mixture of glycerin and water, and keep
                                                the seeds at -80℃.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES3Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES3" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES3Label">ES3</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>After many attempts, we have made some achievements, but some works has
                                                not achieved the desired results, and most of the agarose gel
                                                electrophoresis strips are shorter than the expected length.</p>
                                            <p>It is gratifying that the FMO dimer we built has been successful.</p>
                                            <div class="image"><img
                                                    alt="Fig1. Plasmid construction and colony PCR result of Pynr071c-&#945; factor-FMO dimer-AOX1 Terminator transformed E.coli.The bands of FMO dimer (less than 5000bp) and is identical to the theoretical lengths of 4600bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these target plasmid are successfully constructed."
                                                    src="./statics/pics/Engineering/1.png"
                                                    style="width: 100%" />
                                                <p>Figure 1: Fig1. Plasmid construction and colony PCR result of
                                                    Pynr071c-&#945; factor-FMO dimer-AOX1 Terminator transformed
                                                    E.coli.The bands of FMO dimer (less than 5000bp) and is identical to
                                                    the theoretical lengths of 4600bp estimated by the designed primer
                                                    locations (promoter to terminator), which could demonstrate that
                                                    these target plasmid are successfully constructed.</p>
                                            </div>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES4Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES4" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES4Label">ES4</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>After many attempts and comparing the examples of success and failure, we
                                                conclude that the reasons for failure may come from these two points: 1.
                                                There are a large number of repeated segments on our system. Because all
                                                the promoters and terminators in our device are the same, it means that
                                                our target system will contain a large number of repeated fragments,
                                                which may cause functional confusion in the homologous recombinase.
                                                However, the successful FMO dimer does not contain multiple promoters
                                                and terminators. 2. The principle of the enzyme we use is different from
                                                the classic seamless cloning enzyme.</p>
                                            <p>We came up with two solutions, 1. Replacement of method, using the enzyme
                                                digestion method with a certain success rate to construct pathway. 2.
                                                Replacement of the homologous recombinase.</p>
                                            <p>Combining practical considerations, because we have successfully
                                                constructed the devices using the enzyme digestion method, and
                                                considering the impact of the epidemic on our progress, we chose to
                                                change the construction method, the enzyme digestion method.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <h1>Cycle2:Construct by enzyme digestion and ligation</h1>
                        <p>Classic, reliable but lenthy double-enzyme digestion and reconnection is used to construct
                            basic devices, along with systems with complete function. Eventually, good news is we have
                            succeeded in building all of our basic devices, but bad news is we have no choice but to put
                            off the construction of complete pathways after iGEM, due to the precious but limited time,
                            which undoubtly will be continuously carried forward.</p>
                        <div class="modal-btn"><button class="btn btn-warning pull-right" data-target="#ES5" data-toggle="modal" type="button" style="margin: 0 5px">Design</button><button class="btn btn-warning pull-right" data-target="#ES6" data-toggle="modal" type="button" style="margin: 0 5px">Build</button><button class="btn btn-warning pull-right" data-target="#ES7" data-toggle="modal" type="button" style="margin: 0 5px">Test</button><button class="btn btn-warning pull-right" data-target="#ES8" data-toggle="modal" type="button" style="margin: 0 5px">Learn</button>
                            <div aria-hidden="true" aria-labelledby="ES5Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES5" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES5Label">ES5</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>We utilized the existing components, assemble each component, built the
                                                device, and then connected the devices one by one, stably, got the
                                                system, expecting to achieve our goal.</p>
                                            <p>The first is to construct devices containing multiple single gene
                                                fragments. We first transferred the basic plasmids containing single
                                                gene which were synthesized by the company into E. coli for
                                                amplification. Plasmids containing the promoter Panb1 or Pynr071c became
                                                as vectors after double digestion; PCR was used to obtain the gene
                                                segments, and then double digestion with the same enzyme. The vectors
                                                that had undergone gel extraction and the purified segments are
                                                connected. Finally, the ligated product was transformed into E.coli
                                                TOP10F' and screened by ampicillin 100 ug/mL LB medium. If a single
                                                colony grows, pick the spot and verify the colony by PCR. After
                                                observing the target band, mix the bacterial solution with a 1:1 mixture
                                                of glycerin and water, and keep the seeds at -80℃.</p>
                                            <p>Expand the training of the verified devices, and obtain a large number of
                                                devices needed to build the system.</p>
                                            <p>Secondly, we constructed the system. The plasmid used as the vector is
                                                firstly digested with double enzymes, the fragments were obtained from
                                                previously built devices through PCR. Then the fragments were double
                                                digested with the same enzymes, similarly, the vector needed to undergo
                                                gel extraction. After the PCR fragments were purified, the vector and
                                                the fragments are connected. Same as above, the ligated product was
                                                transformed into E.coli TOP10 F', and then proceeded to screening, PCR
                                                verification and seed preservation in sequence.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES6Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES6" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES6Label">ES6</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>First, double-enzyme digestion of the plasmid as the vector, then use PCR
                                                to obtain the gene segments of each element, then use the same enzymes
                                                to double-enzyme digest the gene segments obtained by PCR, and finally
                                                use ligase to connect the fragments with the vector. Take the
                                                construction of device: Panb1-α factor-crtE-AOX1 Terminator as an
                                                example: firstly, double-enzyme digestion of the vector plasmid
                                                containing the promoter Panb1. Then obtain the fragment from the AOX1-α
                                                factor-crtE-AOX1 Terminator basic plasmid by PCR, and use the same
                                                enzyme double enzyme digest the PCR fragments. After the fragment and
                                                the vector are purified, use Ligation ligase to connect the fragment and
                                                the vector to obtain the device: Panb1-α factor-crtE-AOX1 Terminator.
                                                Then transfer the ligation product to E.coli TOP10 F', and culture it
                                                into LB medium containing ampicillin 100 ug/mL for screening. After a
                                                single colony grows, the spot is picked and verified by PCR. If the
                                                target band is observed, mix the bacterial solution with a 1:1 mixture
                                                of glycerin and water, and keep the seeds at -80℃. The remaining 12
                                                devices are constructed using the same method.</p>
                                            <p>After obtaining all the devices, we constructed the system.
                                                simultaneously using both methods</p>
                                            <p>The 1. multi-step connection.</p>
                                            <p>Double-enzyme digestion of the plasmid as the vector, and then use PCR to
                                                obtain the gene fragments of each device, and then use the same enzyme
                                                to double-enzyme the gene fragments obtained by PCR. After the fragment
                                                and the vector are purified, they are ligated with the vector by ligase.
                                                Repeat the operation until the system is successfully built. Take the
                                                lycopene pathway as an example: First, digest the entire plasmid
                                                containing Panb1-crtE-AOX1 Terminator as a vector by double enzyme
                                                digestion, exposing the sticky end at the back of its gene fragment.
                                                Then obtain the gene fragment of Panb1-crtB-AOX1 Terminator by PCR,
                                                double digesting the gene fragment. After the fragment and vector are
                                                purified, ligase is used to connect with the vector to obtain a new
                                                plasmid: Panb1-crtE-AOX1 Terminator-Panb1-crtI-AOX1 Terminator. Then
                                                transfer the ligation product to E.coli TOP10 F' culture, and inoculate
                                                it into LB medium containing AMP100mg/mL for selection. After a single
                                                colony grows, the spot is picked, and the colony is verified by PCR.
                                                After observing the target band, mix the bacterial solution with a 1:1
                                                mixture of glycerin and water, and keep the seeds at -80℃. Then use the
                                                new plasmid as a vector and Panb1-crtB-AOX1 Terminator as a fragment,
                                                and repeat the operation.</p>
                                            <ol start="2">
                                                <li>One-step connetion</li>
                                            </ol>
                                            <p>The vector and fragment are obtained in the same way. After the fragments
                                                and the vector are purified, ligase is used to ligate multiple fragments
                                                with the vector at once. Take the lycopene pathway as an example:
                                                firstly digest the entire plasmid containing Panb1-crtE-AOX1 Terminator
                                                as a vector, exposing the sticky end at the back of its gene fragment.
                                                Then obtain Panb1-crtB-AOX1 Terminator and Panb1-crtI- by PCR, double
                                                digesting the gene fragment. The purified fragment and vector are
                                                ligated with the vector by ligase to obtain a new plasmid:
                                                Panb1-Lycopene-AOX1 Terminator. The last step is the same
                                                transformation, screening, verification and seed preservation.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES7Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES7" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES7Label">ES7</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>All the constructed devices passed the gel electrophoresis, and all the
                                                respective bands were correct, indicating that the construction was
                                                successful.</p>
                                            <p>Due to time limit, when constructing the system, the Panb1-crtE-AOX1
                                                Terminator-Panb1-crtI-AOX1 Terminator plasmid and FMO duplex were
                                                finally successfully obtained through Method 1 only. The enzyme
                                                digestion method is more complicated and has a long cycle, so it is
                                                temporarily shelved. Method 2 failed after many attempts, and the band
                                                was shorter than expected. It may be that the homologous fragment in the
                                                connecting part of the two fragments is shorter, which affects the
                                                connection.</p>
                                            <div class="image"><img
                                                    alt="Fig2. Plasmid construction and colony PCR results of reconstructed plasmid with Panb1 and Pynr071C promoter.All the bands are identical to the theoretical lengths, which could demonstrate that these plasmid are correctly constructed and successfully transformed into E.coli, confirmed by sequencing."
                                                    src="./statics/pics/Engineering/2.png"
                                                    style="width: 100%" />
                                                <p>Figure 2: Fig2. Plasmid construction and colony PCR results of
                                                    reconstructed plasmid with Panb1 and Pynr071C promoter.All the bands
                                                    are identical to the theoretical lengths, which could demonstrate
                                                    that these plasmid are correctly constructed and successfully
                                                    transformed into E.coli, confirmed by sequencing.</p>
                                            </div>
                                            <div class="image"><img
                                                    alt="Fig3. Plasmid construction and colony PCR result of Panb1-&#945; factor -crtE-AOX1 Terminator-Panb1-&#945; factor -crtI-AOX1 Terminator transformed E.coli.The bands of Panb1-&#945; factor -crtE-AOX1 Terminator-Panb1-&#945; factor -crtI-AOX1 (less than 5000bp) and is identical to the theoretical lengths of 4500bp estimated by the designed primer locations (promoter to terminator), which could demonstrate that these target plasmid are successfully constructed"
                                                    src="./statics/pics/Engineering/3.png"
                                                    style="width: 100%" />
                                                <p>Figure 3: Fig3. Plasmid construction and colony PCR result of
                                                    Panb1-&#945; factor -crtE-AOX1 Terminator-Panb1-&#945; factor
                                                    -crtI-AOX1 Terminator transformed E.coli.The bands of Panb1-&#945;
                                                    factor -crtE-AOX1 Terminator-Panb1-&#945; factor -crtI-AOX1 (less
                                                    than 5000bp) and is identical to the theoretical lengths of 4500bp
                                                    estimated by the designed primer locations (promoter to terminator),
                                                    which could demonstrate that these target plasmid are successfully
                                                    constructed</p>
                                            </div>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES8Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES8" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES8Label">ES8</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>The partial success of Method 1 proved that our design is feasible in
                                                principle, but due to the limited time and the impact of the epidemic,
                                                we temporarily shelved the experiment but this does not mean that we
                                                will give up. After the iGEM finishes, we will continue this project
                                                until the complete system is successfully built. The failure of method 2
                                                is also expected: inserting multiple fragments at once is "bite more
                                                than one can chew", and its success rate is destined to be low. So
                                                method 2 is only an attempt to accumulate experience and train team
                                                members. Scientific research requires our persevering efforts. We insist
                                                on our love and confidence in synthetic biology, and are willing to
                                                contribute to the development of synthetic biology</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <h1>Cycle3:Extracellular expression portion</h1>
                        <p>In order to achieve our purpose of perming and dyeing hair directly, we originally designed
                            to express all enzymes and short peptides outside the cell. So after Electrotransforming the
                            plasmid into yeast, we directly tried to detect our target protein in the supernatant. After
                            centrifugation, we took the supernatant for SDS-PAGE detection. But after many tests, only
                            part of the target protein was detected. After analysis, we suspected that the target
                            protein could not be detected because the extracellular expression was low and unstable, or
                            due to the mismatch between the signal peptide and the spatial structure of the protein. So
                            we want to try express in the cell later.</p>
                        <div class="modal-btn"><button class="btn btn-warning pull-right" data-target="#ES9" data-toggle="modal" type="button" style="margin: 0 5px">Design</button><button class="btn btn-warning pull-right" data-target="#ES10" data-toggle="modal" type="button" style="margin: 0 5px">Build</button><button class="btn btn-warning pull-right" data-target="#ES11" data-toggle="modal" type="button" style="margin: 0 5px">Test</button><button class="btn btn-warning pull-right" data-target="#ES12" data-toggle="modal" type="button" style="margin: 0 5px">Learn</button>
                            <div aria-hidden="true" aria-labelledby="ES9Label" class="modal fade" data-backdrop="static"
                                data-keyboard="false" id="ES9" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES9Label">ES9</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>In order to facilitate the synthesis of pigments and short peptides that
                                                can be directly used for perm and dyeing, we initially chose to express
                                                all enzymes and short peptides extracellularly so that we can use
                                                related enzymes to synthesize pigments in the culture medium and can
                                                directly obtain them for perm and straightening hair. So we added
                                                α-factor as a signal peptide in front of all target genes, in order to
                                                secrete all proteins. At the same time, in order to enhance expression,
                                                we selected the strongest eukaryotic promoter that has been discovered:
                                                AOX1, to initiate and enhance the expression of downstream genes. And in
                                                order to achieve the stable existence of the target gene in yeast and
                                                form a high copy for efficient expression, we selected a suitable
                                                restriction site to linearize the plasmid to integrate into the yeast
                                                genome to ensure that it can remain stable after passaging. And we
                                                expect to achieve high copy through repeated integration. At the same
                                                time, we added His-tag at the end of the gene for protein purification
                                                and identification.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES10Label" class="modal fade"
                                data-backdrop="static" data-keyboard="false" id="ES10" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES10Label">ES10</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>We first transfer the synthesized plasmid into E. coli for amplification
                                                in order to obtain enough target plasmid. We comprehensively considered
                                                the hidden restriction sites of different target genes, and finally
                                                selected two integration sites, BglII and SalI, to avoid the destruction
                                                of all genes. We cut the plasmid with BglII or SalI to obtain a
                                                linearized plasmid. Due to system limitations, we can only divide it
                                                into multiple tubes for single digestion.</p>
                                            <p>Initially, we considered the single enzyme digestion product to be
                                                subjected to agarose gel electrophoresis, and concentration were
                                                performed while verifying. However, in next experiments, we found that
                                                the linear plasmid concentration we obtained was very low. It is
                                                attributed to the lower efficiency of gel extraction and the operation
                                                process: in the last step, repeating the experiment ddH2O elution will
                                                cause a large amount of plasmid-containing solution to remain at the
                                                bottom or wall of the ep tube; in the second elution , we cannot heat
                                                the ddH2O again, and the linearized carrier with a larger molecular
                                                weight is trapped by the purification column with lower elution
                                                efficiency, causing waste. In the end, due to various reasons, the
                                                plasmid concentration was not enough when we first performed the
                                                electroporation. Although part of the linearized vector was successfully
                                                electro-transformed in the follow-up test, and some of the yeast
                                                integrated with the target gene was obtained, the target protein was
                                                almost impossible to be detected in the follow-up identification due to
                                                the low concentration and the failure to form a high copy.</p>
                                            <p>After carefully reviewing the information and asking authoritative
                                                sources, combined with the previous results, we found that at the
                                                concentration of the plasmid we extracted, overnight single enzyme
                                                digestion is completely enough to completely cut all the plasmids. This
                                                means that there are very few unlinearized plasmids in the reaction
                                                system, and there is no need to use electrophoresis to separate them.
                                                Therefore, in the subsequent experiments, we chose the method of ethanol
                                                precipitation to achieve the purification and concentration of the
                                                single enzyme digestion system. After enzyme digestion, only a small
                                                amount of products were taken for agarose gel electrophoresis
                                                verification, and the remaining products were directly used for ethanol
                                                precipitation and concentration, and finally a high concentration of
                                                linearized plasmid was successfully obtained. After electroporation,
                                                screening was performed on MD plates containing 500 mg/mL G418, and more
                                                single yeast colonies integrated with the target gene were obtained, and
                                                expression verification was performed.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES11Label" class="modal fade"
                                data-backdrop="static" data-keyboard="false" id="ES11" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES11Label">ES11</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>We use the colony PCR method to test whether the target gene is
                                                successfully transformed into the yeast cells. The pipette tip of the
                                                picked yeast single colony is lightly dipped in a PCR tube filled with
                                                sterile water, and then filled with 1 mL of antibiotic YPD is cultured
                                                in a 2mL ep tube to realize the amplification of bacterial species while
                                                performing PCR verification, which saves time and reduces the risk of
                                                infection by reducing openings.</p>
                                            <div class="image"><img
                                                    alt="Figure 4 Results of PCR agarose gel electrophoresis of yeast colonies after electroporation.The bright band identified by colony PCR is consistent with the expection, which proves that the target gene was successfully transferred into yeast cells"
                                                    src="./statics/pics/Engineering/4.png"
                                                    style="width: 100%" />
                                                <p>Figure 4: Figure 4 Results of PCR agarose gel electrophoresis of
                                                    yeast colonies after electroporation.The bright band identified by
                                                    colony PCR is consistent with the expection, which proves that the
                                                    target gene was successfully transferred into yeast cells</p>
                                            </div>
                                            <p>After the PCR verification of the colony was correct, we expanded the
                                                culture and induced expression. Since AOX1 is a methanol-inducible
                                                promoter and is strictly inhibited by glycerol, we cultured it in the
                                                initial glycerol-containing medium to expand the number of bacteria, and
                                                then transferred all of its bacteria to a culture medium with methanol
                                                as the sole carbon source. And add 0.5% of the fermentation volume of
                                                methanol every day, and after sampling and centrifugation, the
                                                supernatant was tested by SDS-PAGE. After multiple days of
                                                identification, we successfully detected some target bands. At the same
                                                time, because the molecular weight of PepACS is only 5.08 kDa, it is
                                                almost impossible to separate from bromophenol blue, and its migration
                                                is also greatly affected, so the position of the band is not accurate.
                                                In order to judge whether it was expressed or not, we first filtered the
                                                miscellaneous proteins through 10kDa ultrafiltration tube, then
                                                concentrated it with 3kDa ultrafiltration tube, and then detected it by
                                                SDS-PAGE. Judge whether it is expressed successfully by whether there is
                                                coloring near the bottom line.</p>
                                            <div class="image"><img
                                                    alt="Figure 5 for SDS-PAGE detection of the Laccase GS1154CL LOX2 ACC pepACS DsbC+pepACS supernatant"
                                                    src="./statics/pics/Engineering/5.png"
                                                    style="width: 100%" />
                                                <p>Figure 5: Figure 5 for SDS-PAGE detection of the Laccase GS1154CL
                                                    LOX2 ACC pepACS DsbC+pepACS supernatant</p>
                                            </div>
                                            <p>Laccase GS115 4CL LOX2 ACC pepACS DsbC+pepACS supernatant SDS-PAGE
                                                detection.</p>
                                            <p>Due to glycosylation modification during yeast expression, the apparent
                                                molecular weight will be slightly larger than the molecular weight of
                                                the actual protein. Preliminary detection of Laccase, 4CL, ACC bands
                                                molecular weight is about 75kDa, LOX2 is above 100kDa, DsbC+pepACS is
                                                about 40kDA, slightly larger than its actual molecular weight (Laccase:
                                                57.01kDa, 4CL: 61.88kDa, ACC: 63.40kDa, LOX2: 102.88 kDa, DsbC+pepACS:
                                                31.72 kDa), which are all within the acceptable range, can be considered
                                                to be successfully expressed. At the same time, due to the small
                                                molecular weight of 5-1, it is difficult to separate from bromophenol
                                                blue completely, which means than its migration is to be greatly
                                                affected. But according to the 10kDa lower part of the marker, it can be
                                                seen that it is distinguished from the blank control: there is obvious
                                                coloring, and the successful expression can be confirmed. There are
                                                irregular staining bands on the bromophenol blue line in the PepACS
                                                swimming lane. Although the molecular weight is not accurate, the
                                                successful expression can be judged according to the detection method we
                                                designed.</p>
                                            <p>For the enzymes Laccase and LOX2 whose expression has been confirmed and
                                                have standard enzyme activity determination methods, we tried to
                                                identify their catalytic activity first.</p>
                                            <p>For LOX2: use 27ul linoleic acid, 25ul tween 20, 8ml ddH20, 3M/L NaOH
                                                183ul, and dilute to 50ml as a solution; take 200ul substrate, 1ml
                                                supernatant, and 1ml pH6.0 buffer for measuring absorbance changes at
                                                234nm. And every 0.001 increase in absorbance is regarded as an enzyme
                                                activity unit. Its activity was successfully detected.</p>
                                            <div class="image"><img alt="Figure 6 Enzyme activity determination of LOX2"
                                                    src="./statics/pics/Engineering/6.png"
                                                    style="width: 100%" />
                                                <p>Figure 6: Figure 6 Enzyme activity determination of LOX2</p>
                                            </div>
                                            <p>On SDS-PAGE, Laccase showed a low concentration, so it is not ideal to
                                                directly measure its catalytic activity. We were not sure about its
                                                activity status, thus, we decided to purify it with a nickel column to
                                                increase the concentration of Laccase so as to further detect the enzyme
                                                activity. activity.</p>
                                            <div class="image"><img
                                                    alt="Figure 7c Laccase nickel column purification SDS-PAGE verification"
                                                    src="./statics/pics/Engineering/7.png"
                                                    style="width: 100%" />
                                                <p>Figure 7: Figure 7c Laccase nickel column purification SDS-PAGE
                                                    verification</p>
                                            </div>
                                            <p>It can be seen that there is a clear band at 75kDa, which is consistent
                                                with the band displayed in the supernatant, and the color of bands
                                                darkens as the concentration of the eluent increases, which further
                                                proves that the band we observed before is indeed the target protein
                                                rather than a high-expressed hybrid protein</p>
                                            <p>After obtaining a higher concentration of Laccase after purification, we
                                                mixed 1mL of 1mmol/L ABTS solution with 1mL of supernatant after
                                                centrifugation, and added CuSO4 with a final concentration of 10mmol/L.
                                                The absorbance was measured at 420nm, and it was successfully detected
                                                enzyme activity. On this basis, in order to explore the optimal
                                                conditions for Laccase, we carried out further determinations.</p>
                                            <p>Since the active center of Laccase is copper ion, we first studied the
                                                relationship between the supplemental copper ion concentration and
                                                enzyme activity. The CuSO4 concentration was adjusted to 9mmol/L and
                                                10mmol/L respectively, and the enzyme activity of Laccase was measured
                                                under the same conditions.</p>
                                            <div class="image"><img
                                                    alt="Figure 8 Effects of concentration of CuSO4 on enzyme activity"
                                                    src="./statics/pics/Engineering/8.png"
                                                    style="width: 100%" />
                                                <p>Figure 8: Figure 8 Effects of concentration of CuSO4 on enzyme
                                                    activity</p>
                                            </div>
                                            <p>We found that the enzyme activity is the highest when the final
                                                concentration is 10 mmol. This may be because when the concentration is
                                                too low, it cannot meet the needs of the Laccase active center. And when
                                                the concentration is too high, it affects the binding of Laccase to the
                                                substrate.</p>
                                            <p>We also explored the effect of pH and temperature on Laccase activity.
                                            </p>
                                            <p>pH: The data showed that the optimal pH of Laccse was acidic, so we used
                                                100mM citric acid and sodium citrate to configure pH=3, pH=4.8 and
                                                pH=6.6 buffers respectively. Under the premise that the final
                                                concentration of CuSO4 is still 10mmol/L, 1mL of buffer is added to
                                                control the pH of the reaction system.</p>
                                            <div class="image"><img
                                                    alt="Figure 9 Effects of different pH on enzyme activity"
                                                    src="./statics/pics/Engineering/9.png"
                                                    style="width: 100%" />
                                                <p>Figure 9: Figure 9 Effects of different pH on enzyme activity</p>
                                            </div>
                                            <p>After comparison, we found that among the three sets of data measured,
                                                the activity of Laccase was the highest at pH=3.0. At pH=4.8, the
                                                activity drops sharply, but at pH=6.6, it is completely inactivated. As
                                                for the subsequent measurement value at pH 6.6 presenting lower than the
                                                initial value, we speculated that it is caused by the inadequate mixing
                                                of the solution at the beginning of the reaction.</p>
                                            <p>Temperature: We separately kept the ABTS solution and supernatant at 10℃,
                                                20℃, 37℃ in advance, and also kept the final concentration of CuSO4 at
                                                10mmol/L, and mixed them before the measurement.</p>
                                            <div class="image"><img
                                                    alt="Figure 10 Effects of different temperature on laccase activity"
                                                    src="./statics/pics/Engineering/10.png"
                                                    style="width: 100%" />
                                                <p>Figure 10: Figure 10 Effects of different temperature on laccase
                                                    activity</p>
                                            </div>
                                            <p>The analysis and comparison of the three sets of data shows that under
                                                the test conditions, the Laccase activity is highest at 20℃, and both
                                                low temperature and high temperature will inhibit Laccase activity. As
                                                for the decrease in absorbance at the late of the measurement at 10℃, it
                                                is suspected that the low temperature has a certain effect on the ABTS
                                                cation radicals, resulting in decrease in absorbance.</p>
                                            <p>At the same time, during the detection process, we found that ROX1 has a
                                                relatively vague band, and we cannot confirm its successful expression
                                                based on the protein gel of supernatant. However, ROX1 is a key gene
                                                involved in the regulation of all pathways in our entire design, whose
                                                successful expression is very important to us, so we purified it through
                                                a nickel column for further verification. Fortunately, we purified the
                                                target band and confirmed that it can indeed be expressed.</p>
                                            <div class="image"><img
                                                    alt="Figure 11 SDS-PAGE verification after ROX1 nickel column purification"
                                                    src="./statics/pics/Engineering/11.png"
                                                    style="width: 100%" />
                                                <p>Figure 11: Figure 11 SDS-PAGE verification after ROX1 nickel column
                                                    purification</p>
                                            </div>
                                            <p>SDS-PAGE verification after ROX1 nickel column purification</p>
                                            <p>It can be seen that there is a band at 50-55kDa that is significantly
                                                different from the sample after washing and flow-through treatment. Its
                                                molecular weight is slightly larger than the theoretical value of
                                                47.09kDa, but it can be confirmed within the range of the molecular
                                                weight change caused by glycosylation. ROX1 was successfully expressed.
                                                Although the gradient elution was not performed, there were many
                                                contaminated bands when the target band appeared for the first time, but
                                                according to the subsequent elution results, it can be judged that it is
                                                indeed the target protein.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES12Label" class="modal fade" data-backdrop="static" data-keyboard="false" id="ES12" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES12Label">ES12</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>However, after many times of SDS-PAGE tests, we found that there are
                                                still some proteins that have not been detected in bands, but the
                                                bacterial liquid PCR verification is correct. By analysis, we suspect
                                                that there are two reasons: one is that the extracellular expression of
                                                the protein is low, and it is unstable in the extracellular environment,
                                                and is easily degraded by proteases, etc.; the other is that the signal
                                                peptide we use is a universal signal peptide, absence of particular
                                                design for the target protein, the two do not fit and the protein cannot
                                                be successfully secreted to the outside of the cell.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <h1>Cycle4:Intracellular expression portion</h1>
                        <p>After determining that under the existing system, the extracellular expression of some
                            proteins may not be achieved. We chose to reconstruct the plasmid, remove the signal peptide
                            and try to express it in the cell. So we amplified the target gene sequence without signal
                            peptide by PCR and inserted it behind the Panb1 promoter to construct a plasmid without
                            signal peptide. Also after electrotransformation, we extracted the total protein of yeast,
                            purified it with nickel column and examined it by SDS-PAGE. This time, we successfully
                            detected the rest of our protein and realized the synthesis of indigo and lycopene.</p>
                        <div class="modal-btn"><button class="btn btn-warning pull-right" data-target="#ES13" data-toggle="modal" type="button" style="margin: 0 5px">Design</button><button class="btn btn-warning pull-right" data-target="#ES14" data-toggle="modal" type="button" style="margin: 0 5px">Build</button><button class="btn btn-warning pull-right" data-target="#ES15" data-toggle="modal" type="button" style="margin: 0 5px">Test</button><button class="btn btn-warning pull-right" data-target="#ES16" data-toggle="modal" type="button" style="margin: 0 5px">Learn</button>
                            <div aria-hidden="true" aria-labelledby="ES13Label" class="modal fade"
                                data-backdrop="static" data-keyboard="false" id="ES13" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES13Label">ES13</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>After determining that some proteins cannot be expressed extracellularly
                                                through the existing system, we learned that the expression level of
                                                intracellular expression is much higher than that of extracellular
                                                expression. Intracellular proteins are more stable than they exist
                                                outside the cell, and the problem that the expressed protein cannot be
                                                detected due to signal peptide mismatch is avoided. Although it is more
                                                difficult to detect the protein in the cell, after considering various
                                                factors, we chose to rebuild the plasmid, remove the signal peptide, and
                                                try to express protein in the cell.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES14Label" class="modal fade"
                                data-backdrop="static" data-keyboard="false" id="ES14" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES14Label">ES14</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>We designed new PCR primers on the basis of original sequence to ensure
                                                that the signal peptide is eliminated when the gene is amplified.
                                                However, because the signal peptide part of the plasmid with AOX1 as the
                                                promoter did not have suitable restriction sites, we could not use
                                                restriction enzymes to directly remove them, so we chose the method of
                                                using restriction enzymes to connect the fragments and insert after the
                                                constitutive promoter Panb1. Constitutive promoters can be used to
                                                continuously synthesize the target protein without induction.</p>
                                            <p>In order to ensure the successful construction of the target plasmid, we
                                                first transformed the enzyme-linked product into E. coli for testing and
                                                amplification. After confirming that the target plasmid was obtained, we
                                                still used the same method as before, single enzyme digestion, ethanol
                                                precipitation and concentration, electrotransfection of yeast and
                                                expression.</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES15Label" class="modal fade"
                                data-backdrop="static" data-keyboard="false" id="ES15" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES15Label">ES15</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>In order to confirm the successful construction of the target plasmid, we
                                                first transformed the enzyme ligation product into E.coli, and spread it
                                                on an LB plate containing Kan resistance. Only the E.coli successfully
                                                transferred into the plasmid can grow normally. The colony PCR is used
                                                to identify whether the transferred plasmid is correct, and further
                                                sequence verification is performed to confirm that the plasmid is
                                                successfully constructed without mutation.</p>
                                            <div class="image"><img
                                                    alt="Fig12. Plasmid construction and colony PCR results of plasmid without signal peptides transformed E.coli"
                                                    src="./statics/pics/Engineering/12.png"
                                                    style="width: 100%" />
                                                <p>Figure 12: Fig12. Plasmid construction and colony PCR results of
                                                    plasmid without signal peptides transformed E.coli</p>
                                            </div>
                                            <p>All the bands are identical to the theoretical lengths, which could
                                                demonstrate that these plasmid are correctly constructed and
                                                successfully transformed into E.coli, confirmed by sequencing.</p>
                                            <p>Identical method is used in testing yeast after electroporation: colony
                                                PCR, confirmation, bright bands, expand cultivation.</p>
                                            <div class="image"><img
                                                    alt="Fig13. Colony PCR result of yeast after electroporation of reconstructed plasmid without the signal peptide"
                                                    src="./statics/pics/Engineering/13.png"
                                                    style="width: 100%" />
                                                <p>Figure 13: Fig13. Colony PCR result of yeast after electroporation of
                                                    reconstructed plasmid without the signal peptide</p>
                                            </div>
                                            <p>The bright bands are identical to the theoretical lengths, which could
                                                demonstrate that this target plasmid had successfully transformed into
                                                yeast. Target genes are confirmed exist in the yeast of multiple bands,
                                                which could be the result of polluted electroporation cup.</p>
                                            <p>The constitutive promoter doesn't need to be induced. To save time and
                                                unnecessary steps to reduce the risk of bacterial infection, instead of
                                                changing the culture medium, we culture our yeast continuously in the
                                                same medium with daily supply of glycerol as carbon source.</p>
                                            <p>Total yeast protein extraction kit is used to extract the whole proteome
                                                of our yeast for further test because intracellular expression makes
                                                direct test of target protein in the medium infeasible. Then, we go for
                                                a purification through Nickel-affinity chromatography column, then apply
                                                SDS-PAGE to separate target protein from the large amount and various
                                                type of total protein to confirm the exist of our target protein.</p>
                                            <div class="image"><img
                                                    alt="Fig14. SDS-PAGE result of FMO after purification of yeast total protein extraction product through Nickel-affinity chromatography column"
                                                    src="./statics/pics/Engineering/14.png"
                                                    style="width: 100%" />
                                                <p>Figure 14: Fig14. SDS-PAGE result of FMO after purification of yeast
                                                    total protein extraction product through Nickel-affinity
                                                    chromatography column</p>
                                            </div>
                                            <p>Different from impure or permeate bands, the target protein located
                                                around 60kDa, bigger than the theoretical 53.96kDa but still within
                                                explainable and acceptable range of glycosylation modification. FMO
                                                could be confirmed as successfully expressed. The concentration of yeast
                                                total protein is so high that huge amount of impure protein is included
                                                during elution. But due to difference from impure or permeate bands, its
                                                dark color and consistency among several times of elution, this band
                                                could be verified as our target FMO,</p>
                                            <div class="image"><img
                                                    alt="Fig15. SDS-PAGE result of crtE after purification of yeast total protein extraction product through Nickel-affinity chromatography column"
                                                    src="./statics/pics/Engineering/15.png"
                                                    style="width: 100%" />
                                                <p>Figure 15: Fig15. SDS-PAGE result of crtE after purification of yeast
                                                    total protein extraction product through Nickel-affinity
                                                    chromatography column</p>
                                            </div>
                                            <p>Different from impure or permeate bands, the target protein located
                                                around 50kDa, bigger than the theoretical 33.42kDa but still within
                                                explainable and acceptable range of glycosylation modification. crtE
                                                could be confirmed as successfully expressed.</p>
                                            <div class="image"><img
                                                    alt="Fig16. SDS-PAGE result of crtB after purification of yeast total protein extraction product through Nickel-affinity chromatography column"
                                                    src="./statics/pics/Engineering/16.png"
                                                    style="width: 100%" />
                                                <p>Figure 16: Fig16. SDS-PAGE result of crtB after purification of yeast
                                                    total protein extraction product through Nickel-affinity
                                                    chromatography column</p>
                                            </div>
                                            <p>Different from impure or permeate bands, the target protein located
                                                around 50kDa, bigger than the theoretical 35.30kDa but still within
                                                explainable and acceptable range of glycosylation modification. crtB
                                                could be confirmed as successfully expressed.</p>
                                            <div class="image"><img
                                                    alt="Fig17. SDS-PAGE result of crtI after purification of yeast total protein extraction product through Nickel-affinity chromatography column"
                                                    src="./statics/pics/Engineering/17.png"
                                                    style="width: 100%" />
                                                <p>Figure 17: Fig17. SDS-PAGE result of crtI after purification of yeast
                                                    total protein extraction product through Nickel-affinity
                                                    chromatography column</p>
                                            </div>
                                            <p>Different from impure or permeate bands, the target protein located
                                                around 60kDa, bigger than the theoretical 55.86kDa but still within
                                                explainable and acceptable range of glycosylation modification. crtI
                                                could be confirmed as successfully expressed.</p>
                                            <p>After confirmation of successful expression, we add indole as substrate
                                                into culture medium of FMO and FPP into the mixture of crtE, crtB and
                                                crtI to test whether the expressed enzyme is active in terms of color
                                                change of the culture medium. After culturing for a period of time, blue
                                                and red medium is occurred, which means we succeed in expression of
                                                active enzymes used to synthesize indigo or lycopene. Although the
                                                synthesis level of the pigments is too low to dye, still, it is one huge
                                                step for us.</p>
                                            <div class="image"><img alt="Fig18 Medium for expression with substrates"
                                                    src="./statics/pics/Engineering/18.png"
                                                    style="width: 100%" />
                                                <p>Figure 18: Fig18 Medium for expression with substrates</p>
                                            </div>
                                            <p>From left to right:</p>
                                            <p>GS115 medium with indole and FPP;</p>
                                            <p>Panb1-FMO-AOX1 Terminator medium with indole;</p>
                                            <p>Mixture of Panb1-crtE-AOX1 Terminator??Panb1-crtB-AOX1
                                                Terminator??Panb1-crtI-AOX1 Terminator medium with FPP</p>
                                            <p>Unfortunately, due to limited time, enzymes for synthesis and degradation
                                                of curcumin aren't tested. Neither does the synthesis of curcumin</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="modal-btn">
                            <div aria-hidden="true" aria-labelledby="ES16Label" class="modal fade"
                                data-backdrop="static" data-keyboard="false" id="ES16" tabindex="-1">
                                <div class="modal-dialog modal-dialog-scrollable modal-dialog-centered modal-xl">
                                    <div class="modal-content">
                                        <div class="modal-header">
                                            <h5 class="modal-title" id="ES16Label">ES16</h5>
                                        </div>
                                        <div class="modal-body">
                                            <p>Through intracellular experiment, we confirm that the suitability problem
                                                between the signal peptide and the space structure of our protein do
                                                exist, that is to say, even if the common signal peptide on the most
                                                popular engineering plasmid couldn't fit protein with any kind of space
                                                structure. This part of experiment rescues our experiment result by
                                                expressing enzymes which we could not detect in extracellular expression
                                                and realize the synthesis of pigments primarily.</p>
                                            <p>It must be noted that the expression and synthesis level of our enzymes
                                                and pigments are too low to be applied in actual dyeing due to the
                                                shallow color of supernatant of intracellular expression after
                                                certification. Across all our experiments, we would say it is because
                                                the low copy number of our plasmid within the yeast. Due to limitation
                                                of time and culture scale, we have to pick the relatively big colonies
                                                which grow rapidly as yeast with high copy number. Without rounds of
                                                screening of gradient concentration of antibiotics, our identification
                                                of high copy number is only a suspect and not strict enough, since yeast
                                                with higher copy number will express higher level of antibiotic
                                                resistance product to grow faster in medium containing higher
                                                concentration of antibiotics. So it is possible that the strain we
                                                selected has too low the copy number to express enough amount of target
                                                protein without gradie</p>
                                        </div>
                                        <div class="modal-footer"><button class="btn btn-danger" data-dismiss="modal"
                                                type="button">Close</button></div>
                                    </div>
                                </div>
                            </div>
                        </div>
                        <div class="row g-0">
                            <div class="col-4 girl"><img
                                    src="./statics/pics/Engineering/girl.png" /></div>
                            <div class="col-5">
                                <div class="dialogbox">
                                    <p>I think the methods in constructing engineering plasmids are quite valid, you? Well, I have a question here. What if you or I want to repeat their experiment? I want to know what <a href="Protocol.html">protocols</a> did they follow to get those result.</p>
                                </div>
                            </div>
                        </div>
                    </article>
                </div>
            </div>
        </div>
    </main>
    <footer>
        <div class="container">
            <div class="row">
                <div class="col-7">
                    <div class="row justify-content-center">
                        <div class="footer-logo"></div>
                    </div>
                    <p><span>CONTACT US: </span><a href="mailto:iGEMHUSTChina@163.com">iGEMHUSTChina@163.com</a></p>
                    <p>Huazhong University of Sci. &amp; Tech., Wuhan, China</p>
                    <p>1037# Luoyu Rd, Wuhan, P.R.China 430074</p>
                    <p>Copyright © <span>HUST-China </span>iGEM 2021</p>
                </div>
                <div class="col-5">
                    <div class="row">
                        <div class="footer-xmind"></div>
                    </div>
                    <div class="row">
                        <div class="footer-snapgene"></div>
                    </div>
                    <div class="row">
                        <div class="footer-NEBI"></div>
                    </div>
                </div>
            </div>
        </div>
    </footer>
    <script
        src="https://2021.igem.org/Template:HUST-China/content-bundleJS?action=raw&amp;ctype=text/javascript"></script>
    <script
        src="https://2021.igem.org/Template:HUST-China/mathjax-bundleJS?action=raw&amp;ctype=text/javascript"></script>
</body>


</p>
<!-- 
NewPP limit report
CPU time usage: 0.005 seconds
Real time usage: 0.006 seconds
Preprocessor visited node count: 4/1000000
Preprocessor generated node count: 32/1000000
Post‐expand include size: 0/2097152 bytes
Template argument size: 0/2097152 bytes
Highest expansion depth: 2/40
Expensive parser function count: 0/100
-->

<!-- Saved in parser cache with key 2021_igem_org:pcache:idhash:1486-0!*!*!*!*!*!* and timestamp 20220201200800 and revision id 495843
 -->
</div>		            <div class="visualClear"></div>
	            </div>
		    </div>

        </div>
    </div>
</html>
